npm/private/test/BUILD.bazel

```diff
@@ -30,28 +31,46 @@ yaml_tests(name = "test_yaml")
 
 parse_pnpm_lock_tests(name = "test_parse_pnpm_lock")
 
-generated_pkg_json_test(name = "test_generated_pkg_json")
+# generated_pkg_json_test(name = "test_generated_pkg_json")
```

Comment:
@gregmagolan was this intentional?

Reply:
Whoops. That was commented out during testing and should not have landed. I'll update it.